statistics/handle/handle.go

```diff
@@ -361,10 +364,11 @@ func (h *Handle) MergePartitionStats2GlobalStats(sc *stmtctx.StatementContext, i
 				// If the statistics is the index stats, we should use the index ID to replace the column ID.
 				ID = idxID
 			}
-			hg, cms, topN := partitionStats.GetStatsInfo(ID, isIndex == 1)
+			hg, cms, topN, fms := partitionStats.GetStatsInfo(ID, isIndex == 1)
 			allHg[i] = append(allHg[i], hg)
 			allCms[i] = append(allCms[i], cms)
 			allTopN[i] = append(allTopN[i], topN)
+			allFms[i] = append(allFms[i], fms)
```

Comment:
Please also use them to merge NDV and resolve the TODO in line:413.

Reply:
I will do this in next PR.